src/opnsense/mvc/app/controllers/OPNsense/Firewall/Api/FilterController.php

```diff
@@ -342,7 +342,8 @@ public function getInterfaceListAction()
                 'items' => [
                     [
                         'value' => '',
-                        'label' => gettext('Any')
+                        'label' => gettext('Any'),
+                        'selected' => true
```

Comment:
```suggestion
                        'selected' => true,
```

Reduces the next diff when adding another option ;)